commit 6eea5fbb112f80f8f645295f7294f17a324799da (refs/changes/37/537/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-09-28T10:24:49-07:00 (3 years ago)
    
    TOOLS-1552 Update sdcnode version for OpenSSL 1.0.2j/1.0.1u vulnerability fixes
    Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
    Approved by: Josh Wilsdon <josh@wilsdon.ca>

diff --git a/nodeconfigs.json b/nodeconfigs.json
index 19e08dd..2ae477a 100644
--- a/nodeconfigs.json
+++ b/nodeconfigs.json
@@ -3,7 +3,7 @@
         "// image": "sdc-minimal-multiarch-lts 15.4.1",
         "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
         "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
-        "version": "v4.5.0",
+        "version": "v4.6.0",
         "configure_flags": "--dest-cpu=ia32 --with-dtrace",
         "build_tag": "zone"
     },
@@ -11,11 +11,54 @@
     {
         "// image": "sdc-minimal-multiarch-lts 15.4.1",
         "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
-        "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
-        "version": "v4.4.7",
+        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
+        "version": "v0.12.16",
+        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
+        "build_tag": "zone"
+    },
+    {
+        "// image": "sdc-multiarch 13.3.1",
+        "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
+        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
+        "version": "v0.12.16",
         "configure_flags": "--dest-cpu=ia32 --with-dtrace",
         "build_tag": "zone"
     },
+    {
+        "// image": "sdc-multiarch 13.3.1",
+        "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
+        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
+        "version": "v0.12.16",
+        "configure_flags": "--dest-cpu=x64 --with-dtrace",
+        "build_tag": "zone64"
+    },
+    {
+        "// image": "sdc-smartos 1.6.3",
+        "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
+        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
+        "version": "v0.12.16",
+        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
+        "build_tag": "zone"
+    },
+    {
+        "// image": "sdc-smartos 1.6.3",
+        "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
+        "comment": "For use by agents in the GZ. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
+        "version": "v0.12.16",
+        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
+        "FUTURE_configure_flags": "--with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
+        "build_tag": "gz"
+    },
+    {
+        "// image": "sdc-base@14.2.0",
+        "image": "de411e86-548d-11e4-a4b7-3bb60478632a",
+        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
+        "version": "v0.12.16",
+        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
+        "build_tag": "zone"
+    },
 
     {
         "// image": "sdc-minimal-multiarch-lts 15.4.1",
@@ -157,6 +200,62 @@
         "build_tag": "zone"
     },
 
+    {
+        "// image": "sdc-multiarch 13.3.1",
+        "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
+        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
+        "version": "v0.10.47",
+        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
+        "build_tag": "zone"
+    },
+    {
+        "// image": "sdc-multiarch 13.3.1",
+        "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
+        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
+        "version": "v0.10.47",
+        "configure_flags": "--dest-cpu=x64 --with-dtrace",
+        "build_tag": "zone64"
+    },
+    {
+        "// image": "sdc-smartos 1.6.3",
+        "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
+        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
+        "version": "v0.10.47",
+        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
+        "build_tag": "zone"
+    },
+    {
+        "// image": "sdc-smartos 1.6.3",
+        "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
+        "comment": "For use by agents in the GZ. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
+        "version": "v0.10.47",
+        "configure_flags": "--dest-cpu=ia32 --with-dtrace",
+        "FUTURE_configure_flags": "--with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
+        "build_tag": "gz"
+    },
+    {
+        "// image": "sdc-base@14.2.0",
+        "image": "de411e86-548d-11e4-a4b7-3bb60478632a",
+        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
+        "version": "v0.10.47",
+        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
+        "build_tag": "zone"
+    },
+    {
+        "// image": "sdc-base@14.2.0",
+        "image": "de411e86-548d-11e4-a4b7-3bb60478632a",
+        "comment": "For use by services running in zones. Plus the DOCKER-515 ECDH cipher support for node 0.10.",
+        "version": "v0.10.47",
+        "patch_files": [
+            "patches/ecdh-ciphers-support-v0.10.patch"
+        ],
+        "configure_flags": "--dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
+        "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
+        "build_tag": "zoneecdh"
+    },
+
     {
         "// image": "sdc-multiarch 13.3.1",
         "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
